parsec/core/fs/workspacefs/file_transactions.py

```diff
@@ -50,7 +50,7 @@ def padded_data(data: bytes, start: int, stop: int) -> bytes:
     if start <= stop <= 0:
         return b"\x00" * (stop - start)
     if 0 <= start <= stop:
-        return data[start:stop]
+        return bytes(data)[start:stop]
```

Comment:
why this is needed ? can't we do like what is done in HashDigest binding instead ?

https://github.com/Scille/parsec-cloud/blob/24a2851b33063d55f2ce956c0decf52f06f199ac/oxidation/libparsec_python/src/crypto.rs#L24-L31